introduction.tex

```diff
@@ -48,15 +48,21 @@ \subsection{Structure}
 
 \subsection{Register Definition Format}
 
-All register definitions in this document follow the format shown below.
-A simple graphic shows which fields are in the register. The
-upper and lower bit indices are shown to the top left and top right of each
-field. The total number of bits in the field are shown below it.
+All register definitions in this document follow the format shown below.  A
+simple graphic shows which fields are in the register. The upper and lower bit
+indices are shown to the top left and top right of each field. The total number
+of bits in the field are shown below it.
 
 After the graphic follows a table which for each field lists its name,
 description, allowed accesses, and reset value. The allowed accesses are listed
 in Table~\ref{tab:access}.
 
+Names of registers and their bits are hyperlinks to their definition. If you're
```

Comment:
"and their bits" -> "and their fields"

Reply:
suggest: "Names of registers and their fields are hyperlinks to their definition, and are indexed on page \pageref{index}".

Reply:
Done.